activity.js

```diff
@@ -2,35 +2,34 @@ document.addEventListener('DOMContentLoaded', function () {
 
   chrome.storage.sync.get(["contrast_service_key", "teamserver_url"], function (items) {
 
-
     if (items["contrast_service_key"] != null) {
 
       getActivities(function () {
         return function (e) {
-          xhr = e.currentTarget
+          xhr = e.currentTarget;
           if (xhr.readyState == 4) {
             if (xhr.status == 403) {
               //Configuration problem
-              $("body").addClass("configuration-problem")
-              $("#configuration-problem").show()
+              $("body").addClass("configuration-problem");
+              $("#configuration-problem").show();
             } else if (xhr.responseText == "") {
-              $("body").addClass("no-activity")
-              $("#get-started").show()
+              $("body").addClass("no-activity");
```

Comment:
Wouldn't be easier for you if you made this strings into constants?

`const BODY_CONST = "body";`
`$(BODY_CONST).addClass("no-activity");`

Even the autocomplete on IDEs or code editors would help you more and if you required to change the text, it would be easier.

Reply:
Replaced with constant